modules/shared/src/main/scala/org/tessellation/currency/schema/FeeTransaction.scala

```diff
@@ -0,0 +1,22 @@
+package org.tessellation.currency.schema
+
+import org.tessellation.ext.derevo.ordering
+import org.tessellation.schema.address.Address
+import org.tessellation.schema.balance.Amount
+import org.tessellation.schema.transaction.{TransactionOrdinal, TransactionReference, TransactionSalt}
+
+import derevo.cats.{order, show}
+import derevo.circe.magnolia.{decoder, encoder}
+import derevo.derive
+
+@derive(decoder, encoder, order, ordering, show)
+case class FeeTransaction(
+  source: Address,
+  destination: Address,
+  amount: Amount,
+  parent: TransactionReference,
+  salt: TransactionSalt
```

Comment:
I wonder if we need it for `FeeTransaction`. I actually don't remember if it was necessary for regular transaction in the end 🤔